audit_trail_integration_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build integration
+// +build integration
+
+package tfe
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAuditTrailRead(t *testing.T) {
+	skipIfFreeOnly(t)
```

Comment:
Audit Trails are not available in TFE - I think we have a skipIfEnterprise?

Reply:
Done 👍